caffe2/core/operator_schema.h

```diff
@@ -478,9 +478,9 @@ inline vector<TIndex> GetDimsVector(const TensorShape& shape) {
 }
 
 // Helper function
-inline uint64_t nElemFromDim(const TensorShape& X) {
-  uint64_t nElem = X.dims_size() > 0 ? 1 : 0;
-  for (int i = 0; i < X.dims_size(); ++i) {
+inline uint64_t nElemFromDim(const TensorShape& X, int dim = 0) {
+  uint64_t nElem = 1;
```

Comment:
Are we dropping the `nElem == 0` case?

Reply:
Yes, to simplify code based on discussion with Alexander.